src/components/layout/nav/signup/SignUp.jsx

```diff
@@ -9,11 +10,12 @@ const SignUp = () => {
   return (
     <>
       <Col xl={11} className="signupBox">
-        <GoogleAndFacebookLogIn />
         <Login />
         <Col className="signupRedirect">
-          <NavLink to="/signup">Still not a user? Sign up now!</NavLink>
+          <NavLink to="/signup">Still not a user? Register now!</NavLink>
         </Col>
+        <Row className="socialMediaSeparator"></Row>
```

Comment:
Why did u use an empty <Row> in here , and the parent is  <Col> without any <Container> or <Row> ?

Reply:
it's separator line with a text that is centered, you can check it on netlify deploy.

Reply:
We don't have to use .containrr and .row wrapper just to use .col, right? Same goes here too, is it wrong?